src/features/commands/poll.js

```diff
@@ -0,0 +1,183 @@
+                        .setDescription('Second choice.')
+                        .setRequired(true))
+                .addStringOption(option =>
+                    option.setName('choice-c').setDescription('Third choice.'))
+                .addStringOption(option =>
+                    option.setName('choice-d').setDescription('Fourth choice.'))
+                .addStringOption(option =>
+                    option.setName('choice-e').setDescription('Fifth choice.')))
+        .addSubcommand(subcommand =>
+            subcommand
+                .setName('list')
+                .setDescription('Lists polls that were created in this server.'))
+        .addSubcommand(subcommand =>
+            subcommand
+                .setName('results')
+                .setDescription('Displays results of a poll.')
+                .addStringOption(option => option.setName("poll-id").setDescription("The poll ID to get results from.").setRequired(true))),
+    async execute(client, interaction) {
+        const pollTitle = interaction.options.getString("title");
+        const pollChoices = [interaction.options.getString("choice-a"), interaction.options.getString("choice-b"), interaction.options.getString("choice-c"), interaction.options.getString("choice-d"), interaction.options.getString("choice-e")].filter((a) => a);
+
+        if (!interaction.guild) {
+            interaction.reply({ content: 'You can only use this command in servers!', ephemeral: true });
+            return;
+        }
+
+        switch (interaction.options._subcommand) {
+            case "create":
+                const multipleAnswersAllowed = interaction.options.getBoolean('allow-multiple-answers', true);
+                strawpoll.createStrawpoll(pollTitle, pollChoices, multipleAnswersAllowed).then(response => {
+                    db.createPoll(interaction.member.user.id, interaction.guild.id, `${pollTitle}:${response.pollId}`).then(resp => {
+                        const publicPollEmbed = new MessageEmbed()
+                            .setColor("#0099ff")
+                            .setTitle(`Poll: ${pollTitle}`)
+                            .addField("Choices", pollChoices.map(choice => `⦿ ${choice}: 0`).join('\n'))
+                            .setFooter(`Poll ID: ${response.pollId}`);
+
+                            const Buttons = new MessageActionRow()
+                                .addComponents(
+                                    new MessageButton()
+                                        .setLabel(`Vote`)
+                                        .setURL(`https://strawpoll.com/${response.pollId}`)
+                                        .setStyle('LINK')
+                                )
+
+                        interaction.reply({ embeds: [publicPollEmbed], components: [Buttons] });
+                    });
+                    setInterval(() => updateResults(pollTitle, response.pollId, interaction), 60000);
```

Comment:
This is an inefficient way to do this
```suggestion
//                    setInterval(() => updateResults(pollTitle, response.pollId, interaction), 60000);
```